apps/vc-api/src/config/env-vars-validation-schema.ts

```diff
@@ -8,5 +8,9 @@ import * as Joi from 'joi';
 export const envVarsValidationSchema = Joi.object({
   NODE_ENV: Joi.string().valid('development', 'production', 'test').default('development'),
   PORT: Joi.number().integer().positive().default(3000),
-  BASE_URL: Joi.string().uri().default('http://localhost:3000')
-});
+  BASE_URL: Joi.string().uri().default('http://localhost:3000'),
+  ASKAR_LABEL: Joi.string().default('vc-api-agent'),
+  ASKAR_WALLET_ID: Joi.string().default('vc-api-wallet'),
+  ASKAR_WALLET_KEY: Joi.string().default('vc-api-wallet-key-0001'),
+  ASKAR_WALLET_DB_TYPE: Joi.string().default('vc-api-agent'),
```

Comment:
It would be better if this was a valid `DB_TYPE`
```suggestion
  ASKAR_WALLET_DB_TYPE: Joi.string().default('sqlite'),
```